compiler/rustc_const_eval/src/interpret/operator.rs

```diff
@@ -330,7 +330,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             _ if left.layout.ty.is_any_ptr() => {
                 // The RHS type must be the same *or an integer type* (for `Offset`).
```

Comment:
Please update this comment, adding a link to https://github.com/rust-lang/rust/issues/91636.